modules/docker/README.md

```diff
@@ -8,16 +8,20 @@ Defines [Docker][1] aliases and functions.
 
 - `dk` is short for `docker`
 - `dka` Attach to a running container
+- `dka!` Attach to a running container by name
 - `dkb` Build an image from a Dockerfile
 - `dkd` Inspect changes on a container's filesystem
 - `dkdf` Show docker filesystem usage
 - `dke` Run a command in a running container
 - `dkE` Run an interactive command in a running container
+- `dkE!` Run an interactive command in a running container by name
```

Comment:
Adding a `!` in an alias for an optional parameter seems little non-standard. How about replacing `dkE` alias with a function that takes an optional name?

Reply:
You're right. These new bang aliases are not mature yet. I've removed them from this PR. I'll submit something more robust and well named later on ;)